sdk/program/src/instruction.rs

```diff
@@ -215,6 +215,8 @@ pub enum InstructionError {
     /// Unsupported sysvar
     #[error("Unsupported sysvar")]
     UnsupportedSysvar,
+    // Note: For any new error added here an equivilent ProgramError and it's
+    // conversions must also be added
```

Comment:
Do new errors need featurization or just modification to existing ones?

Reply:
New errors that get surfaced in the final transaction error code need featurization since they get included in the bash hash for that block

Reply:
True for program errors because programs can return whatever they want.  But not true if just an instruction error is added if the use of that instruction error by the runtime is featurized.  I'll make that more clear in the comment.

Reply:
Thanks.  It's definitely subtle